### ISSUE TITLE
Syntax cooiring for java help buffers

### ISSUE THREAD

**Issue by alikthename** [emacs-lsp/lsp-ui#689]
When calling lsp-describe-thing-at-point the buffer with javadoc opens up.
Everything is good there except the code examples are all with one color (no syntax coloring).

**Comment by LionyxML** [comment]
@jcs090218  by any means did you managed to fix it?

I am having the same issue right now with typescript/rust or any other.

Also happens on eldoc.

**Comment by jcs090218** [comment]
Yeah, it should have been fixed a long time ago. I'm unsure about the `eldoc`, but the `lsp-ui-doc` should be fixed. 🤔

**Comment by LionyxML** [comment]
Probably the case.

I opened an issue on markdown-mode package, lets see if it gets some attention:  https://github.com/jrblevin/markdown-mode/issues/824
